### PR TITLE
test: converting a portion of tests from easymock to mockito

### PR DESCRIPTION
**Pull request by dmitry-fa**
another portion of tests from EasyMock to Mockito

### CODE REVIEW

**Review by athakor — APPROVED** [review]
LGTM

File: google-cloud-storage/src/test/java/com/google/cloud/storage/StorageImplMockitoTest.java [hunk 1]
```diff
@@ -458,6 +459,191 @@ public void testCreateBucketFailure() {
...
+    initializeService();
+    Blob blob =
+        storage.get(
+            BLOB_INFO1.getBlobId(), BLOB_GET_METAGENERATION, BLOB_GET_GENERATION_FROM_BLOB_ID);
+    assertEquals(expectedBlob1, blob);
+  }
+
+  @Test
+  public void testGetBlobWithSelectedFields() {
+    ArgumentCaptor<Map<StorageRpc.Option, Object>> capturedOptions =
```
Comment (pmakani) [inline google-cloud-storage/src/test/java/com/google/cloud/storage/StorageImplMockitoTest.java:575]: you can define argumentcapture at global level so you don't need to declare in every test case

**Review by pmakani — APPROVED** [review]
LGTM except that argumentcapture if you want to remove extra lines of code

File: google-cloud-storage/src/test/java/com/google/cloud/storage/StorageImplMockitoTest.java (hunk 1 above)
Comment (dmitry-fa) [reply google-cloud-storage/src/test/java/com/google/cloud/storage/StorageImplMockitoTest.java:575]: Thanks for the suggestion. argumentCapture is used not in all cases, but in a few ones. If we declare it globally we have to clean it up in setUp() method which is runnig before each test.  Other tests use other captures, so I think I will leave this as is.

**Review by rahulKQL — APPROVED** [review]

**Pull request closed without merging**